hummingbot/connector/exchange/bittrex/bittrex_exchange.py

```diff
@@ -0,0 +1,400 @@
+            exchange_order_id=exchange_order_id,
+            trading_pair=tracked_order.trading_pair,
+            update_timestamp=update_time,
+            new_state=new_state,
+        )
+        return update
+
+    async def _user_stream_event_listener(self):
+        async for stream_message in self._iter_user_event_queue():
+            try:
+                content = stream_message.get("delta") or stream_message.get("deltas")
+                if isinstance(content, List):
+                    safe_ensure_future(self._process_execution_event(content))
+                elif "marketSymbol" not in content:
+                    asset_name = content["currencySymbol"]
+                    total_balance = Decimal(content["total"])
+                    available_balance = Decimal(content["available"])
+                    self._account_available_balances[asset_name] = available_balance
+                    self._account_balances[asset_name] = total_balance
+                else:
+                    safe_ensure_future(self._process_order_update_event(content))
+            except asyncio.CancelledError:
+                raise
+            except Exception:
+                self.logger().error("Unexpected error in user stream listener loop.", exc_info=True)
+
+    async def _process_order_update_event(self, msg: Dict[str, Any]):
+        order_id = msg["id"]
+        update_time = _get_timestamp(msg["updatedAt"])
+        order_state = self._get_order_status(msg)
+        tracked_order = None
+        for order in self._order_tracker.all_updatable_orders.values():
+            if order.exchange_order_id is None:
+                continue
+            exchange_order_id = await order.get_exchange_order_id()
+            if exchange_order_id == order_id:
+                tracked_order = order
+                break
+        if tracked_order is not None:
+            order_update = OrderUpdate(
+                trading_pair=tracked_order.trading_pair,
+                update_timestamp=update_time,
+                new_state=order_state,
+                client_order_id=tracked_order.client_order_id,
+                exchange_order_id=order_id,
+            )
+            self._order_tracker.process_order_update(order_update=order_update)
+
+    async def _process_execution_event(self, events: Dict[str, Any]):
+        for execution_event in events:
+            order_id = execution_event["orderId"]
+            tracked_order = None
+            for order in self._order_tracker.all_fillable_orders.values():
+                if order.exchange_order_id is None:
+                    continue
+                exchange_order_id = await order.get_exchange_order_id()
+                if exchange_order_id == order_id:
+                    tracked_order = order
+                    break
+            if tracked_order is not None:
+                percent_token = split_hb_trading_pair(tracked_order.trading_pair)[-1]
```

Comment:
Please use the code that relies on the symbol to trading pair map to get the trading pair

Reply:
Have used the existing quote asset property.